src/main/java/io/leonis/zosma/ipc/peripheral/XboxController.java

```diff
@@ -0,0 +1,32 @@
+package io.leonis.zosma.ipc.peripheral;
+
+import io.leonis.zosma.game.Identity;
+import io.leonis.zosma.ipc.peripheral.XboxController.XboxControllerIdentity;
+import io.leonis.zosma.ipc.peripheral.component.*;
+import lombok.Value;
+
+/**
+ * The Class XboxController.
+ *
+ * Describes the state of the buttons in a xbox controller.
+ *
+ * TODO: This should be moved to Subra
+ * @author Jeroen de Jong
+ */
+@Value
+public class XboxController implements Gamepad<XboxControllerIdentity>, Xbox.Supplier {
+
+  private final String name;
+  private final XboxControllerIdentity identity;
+
+  private final Dpad dpad;
+  private final Stick leftStick, rightStick;
+  private final boolean aPressed, bPressed, xPressed, yPressed;
+  private final boolean leftBumperPressed, rightBumperPressed, backPressed, startPressed;
+  private final float leftTrigger, rightTrigger;
+
+  @Value
+  class XboxControllerIdentity implements Identity {
```

Comment:
can be renamed to Identity?

Reply:
I rather use the same naming as we do for `Player`, e.g. `PlayerIdentity`. If you have a different opinion we could reevaluate that decision.

Reply:
you're right. disregard my suggestion